packages/callout/src/it-callout-more.ts

```diff
@@ -0,0 +1,49 @@
+import { BaseComponent } from '@italia/globals';
+import { html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import styles from './callout-more.scss';
+import './it-callout-more-collapse.js';
```

Comment:
Sospetto che questo crei un import doppio per un utente finale, lo testo anche io comunque intanto.
Il motivo è che it-callout-more-collapse è esportato anche da index e il custom element viene definito come side effect e quindi eseguito sia qui che in index.